src/stories/RefAnchoredOverlay.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import React, {useState, useRef} from 'react'
+import {Button, Text, ButtonDanger, Position, Flex, BaseStyles, ThemeProvider} from '..'
+import {Meta} from '@storybook/react'
+
+import {AnchoredOverlay} from '../AnchoredOverlay'
+
+export default {
+  title: 'Internal components/AnchoredOverlay',
+  component: AnchoredOverlay,
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Story />
+          </BaseStyles>
+        </ThemeProvider>
+      )
+    }
+  ]
+} as Meta
+
+export const RefAnchoredOverlay = () => {
```

Comment:
This is a better/more direct example.  Since we no longer have the `RefAnchoredOverlay`, it probably makes sense to move this into one of the other `.stories` files.  I'd suggest switching this to use `ActionMenu` and renaming it to `ActionMenuWithExternalAnchor` or something like that.